src/core/tsi/alts_transport_security.cc

```diff
@@ -28,38 +28,66 @@ alts_shared_resource* alts_get_shared_resource(void) {
   return &g_alts_resource;
 }
 
-static void grpc_tsi_alts_wait_for_cq_drain() {
-  gpr_mu_lock(&g_alts_resource.mu);
-  while (!g_alts_resource.is_cq_drained) {
-    gpr_cv_wait(&g_alts_resource.cv, &g_alts_resource.mu,
+static void wait_for_cq_drain() {
+  while (!g_alts_resource.is_cq_drained)
```

Comment:
Please put braces around the body of this loop.

Reply:
Done.